conans/test/util/tools_test.py

```diff
@@ -135,6 +135,8 @@ def system_package_tool_fail_when_not_0_returned_test(self):
 
     def system_package_tool_test(self):
 
+        os.environ["CONAN_SYSREQUIRES_SUDO"] = "True"
```

Comment:
```python
with tools.environment_append({"CONAN_SYSREQUIRES_SUDO": "True"}):
     #task here
```
is a context that makes sure that the environment ends clean, even if the tests fails or is broken.